dev-duckies-sm-proj/src/app-wide-components/post.jsx

```diff
@@ -0,0 +1,29 @@
+const Post = (props) => {
+  return (
+    <div className="box">
+      <div className="media m-auto">
+        <div className="media-left">
+          <figure className="image is-48x48">
+            <img
+              className="is-rounded"
+              src="https://bulma.io/assets/images/placeholders/96x96.png"
+              alt="Placeholder image"
+            />
+          </figure>
+        </div>
+        <div className="media-content">
+          <p className="title is-4">{props.username}</p>
```

Comment:
Here it is `username` but in the `public-feed.jsx` the `<Post />` component property for the username is `user`. I recommend updating the <Post /> component's properties to `username` so it displays the username instead of nothing.